### PR TITLE
[ART-7053] Use JIRA field names

### PR DESCRIPTION
**Pull request by ashwindasr**
Ref https://issues.redhat.com/browse/ART-7053

Pulling the field ID using JIRA API since customfield IDs can change without notice

### CODE REVIEW

**Comment by ashwindasr** [comment]
/hold

Until tested

File: elliott/elliottlib/bzutil.py [hunk 1]
```diff
@@ -584,15 +584,34 @@ def get_flaw_bugs(self, bug_ids: List, strict: bool = True, verbose: bool = Fals
         raise NotImplementedError
 
 
+def get_jira_name_id_mapping() -> dict:
+    """
+    Assuming that no two JIRA fields have the same name
+    """
+    result_json = requests.get(constants.JIRA_API_FIELD).json()
```
Comment (joepvd) [inline elliott/elliottlib/bzutil.py:591]: Would not mind a retry here for the web request.

Comment (ashwindasr) [reply elliott/elliottlib/bzutil.py:593]: added in https://github.com/openshift-eng/art-tools/pull/499/commits/28f35844d0de48d65dfff4642e498da1d97ce488

**Comment by ashwindasr** [comment]
/unhold

Tested with `elliott -g openshift-4.16 find-bugs:qe
`

**Comment by Ximinhan** [comment]
nice!
/lgtm

**Pull request merged**